packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1001,6 +1001,13 @@ export const MissingLocale = {
 		`The locale/path \`${locale}\` does not exist in the configured \`i18n.locales\`.`,
 } satisfies ErrorData;
 
+export const MissingIndexForInternalisation = {
```

Comment:
Not specifically related to this PR but shouldn't it be internationalization? AFAIK z is more American and s more British. I'm quite curious!